app_simple.py

```
@@ -0,0 +1,481 @@
+            'function_count': len(functions),
+            'avg_variable_length': np.mean([len(v) for v in variables]) if variables else 0,
+            'avg_function_length': np.mean([len(f) for f in functions]) if functions else 0
+        }
+    
+    def _analyze_complexity(self, code):
+        """Analyze code complexity"""
+        try:
+            tree = ast.parse(code)
+            complexity_metrics = self._calculate_cyclomatic_complexity(tree)
+            return complexity_metrics
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)